include/vctr/Containers/VctrBase.h

```diff
@@ -1091,6 +1091,9 @@ class VctrBase : private Config,
     template <class T>
     VCTR_FORCEDINLINE constexpr static T* assumeAlignedToMaxSIMDRegisterSize (T* ptr)
     {
+        if (std::is_constant_evaluated())
+            return ptr;
+
         #if __cpp_lib_assume_aligned
```

Comment:
Not new code but this is a rather unusual indentation for a preprocessor directive, no?

Reply:
Indeed! I'll fix that